.github/workflows/pull-request-check.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        node-version: [12, 14, 16]
+        node-version: [12, 14, 17]
```

Comment:
Let's create a ticket explaining the bug in Node 16. I think this PR might have to be pending that release, let's discuss with @eaviles  😔

Reply:
A new issue has been created, it seems like they have cherry picked this fix for the next Node v16 release. Ref: https://github.com/nodejs/node/issues/40663